documentation/modules/con-standard-infra-config.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// assembly-plans.adoc
+
+[id='con-standard-infra-config-{context}']
+= Standard infrastructure configuration
+
+`StandardInfraConfig` resources is used to configure infrastructure deployed by `standard` address
+spaces. The standard infra configuration is referenced by address space plans using a `defined-by`
```

Comment:
Change "infra" to "infrastructure"
Also see earlier comment/question about adding the word "file"